source/registry/package.json

```diff
@@ -20,7 +20,6 @@
     "typechain": "tsc -b",
     "coverage": "hardhat coverage",
     "test": "hardhat test",
-    "test:unit": "hardhat test ./test/unit.js",
```

Comment:
`test:unit` is called by CI so let's just update the file name here to test/Registry.js.

Reply:
That won't run test/RegistryIntegration.js. Is that ok?

Reply:
Good point, updated here: https://github.com/airswap/airswap-protocols/pull/1176

Reply:
@dmosites I believe this is not resolved. Correct? `"test": "hardhat test",` will run both Registry.js and RegistryIntegration.js.

Let me know if there's anything else to be done on that.